### PR TITLE
3 19 release notes

### PR DESCRIPTION
**Pull request by lzdanski**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,45 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+:::
+
+In addition to exporting report data with downloads or email, you can now [export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) using webhooks, so you can send your Astronomer data to your webhook endpoints such as Segment, Airtable, or Marketo.
+
+Note that Webhook exports is a [Sigma beta feature](https://help.sigmacomputing.com/docs/webhook-exports) and might experience behavior changes.
+
+### Additional improvements
+
+- Added a new **Credits** tab to the [Organization Billing page](https://docs.astronomer.io/astro/manage-billing), so you can see your credit balance.
+- For Astro Hybrid, the maximum worker concurrency increased from `64` to `256`.
```
Comment (conorbev) [inline astro/release-notes.md:60]: There are also changes here for Hosted

here is the current Hosted UI in prod:

image


^ it's a drop down

This has been changed to an entry:

image

**Review by conorbev — APPROVED** [review]

File: astro/release-notes.md (hunk 1 above)
Comment (lzdanski) [reply astro/release-notes.md:60]: Update from slack discussion - will not add at this time. UI design is sufficient to communicate this change.

File: astro/resource-reference-hosted.md [hunk 2]
```diff
@@ -74,31 +74,33 @@ Currently, standard clusters are available on the following clouds and regions:
...
 <TabItem value="azure">
 
-| Code         | Region                    |
-| ------------ | ------------------------- |
-| `eastus2`    | Virginia, North America   |
-| `westus2`    | Washington, North America |
-| `westeurope` | Netherlands, Europe       |
+| Code             | Region                    |
+| ---------------- | ------------------------- |
+| `centralus`      | Iowa, North America       |
```
Comment (chomipi88) [inline astro/resource-reference-hosted.md:79]: New regions are not available in Standard... it's for Azure Dedicated only.

File: astro/resource-reference-hosted.md [hunk 3]
```diff
@@ -74,31 +74,33 @@ Currently, standard clusters are available on the following clouds and regions:
...
-        {label: 'AWS', value: 'aws'},
-        {label: 'GCP', value: 'gcp'},
-        {label: 'Azure', value: 'azure'},
-    ]}>
+defaultValue="aws"
+groupId= "dedicated-cluster-regions"
+values={[
+{label: 'AWS', value: 'aws'},
+{label: 'GCP', value: 'gcp'},
+{label: 'Azure', value: 'azure'},
```
Comment (chomipi88) [inline astro/resource-reference-hosted.md:102]: new Azure regions are missing here

File: astro/release-notes.md [hunk 4]
```diff
@@ -21,6 +21,45 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+See [Astro Hosted resource reference](https://docs.astronomer.io/astro/resource-reference-hosted) for more information.
+
+### Additional Deployment-level permissions customization
+
+:::publicpreview
+:::
+
+The ability to customize Deployment-level permissions with Deployment roles is now in [Public Preview](https://docs.astronomer.io/astro/feature-previews).
+
+In addition to this change, you can now use the new roles **Workplace accessor** and **Deployment Admin** to refine how you control who has access to your Deployments. See more in [User permissions reference](https://docs.astronomer.io/astro/user-permissions) and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
```
[the remark this answers is not in the document]
Comment (lzdanski) [reply astro/release-notes.md]: If you click these links, they take you to what is live, that content is still WIP in https://github.com/astronomer/docs/pull/3532/files

[the remark this answers is not in the document]
Comment (lzdanski) [reply astro/release-notes.md]: - [Workspace Accessor](https://deploy-preview-3532--gallant-neumann-56599d.netlify.app/astro/user-permissions#workspace-roles) (adding it to the table will break the formatting to add one row of information, so I'm making the judgement call that having it as a bullet point is worth the trade-off for consistency)
- [Deployment Admin](https://deploy-preview-3532--gallant-neumann-56599d.netlify.app/astro/user-permissions#deployment-admin-role-public-preview)

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 5]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## March 19, 2024
+
+### New Astro Hosted regions added for Azure
```
Comment (jwitz) [inline astro/release-notes.md:26]:
```suggestion
### New Azure regions available on Astro Hosted
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+The ability to customize Deployment-level permissions with Deployment roles is now in [Public Preview](https://docs.astronomer.io/astro/feature-previews) for Enterprise tier users.
+
+In addition to this change, you can now use the new roles, **Workplace Accessor** and **Deployment Admin** to refine how you control who has access to your Deployments. See more in the [User permissions reference](https://docs.astronomer.io/astro/user-permissions#workspace-roles), [Deployment Admin](https://docs.astronomer.io/astro/user-permissions#deployment-admin), and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
+
+New Asro CLI commands enable you to use the CLI to interact with and update your users' Deployment roles with the following commands:
+
+* [`astro organization role list`](https://docs.astronomer.io/astro/cli/astro-organization-role-list)
+* [`astro deployment user`](https://docs.astronomer.io/astro/cli/astro-deployment-user)
+* [`astro deployment team`](https://docs.astronomer.io/astro/cli/astro-deployment-team)
+
```
Comment (jwitz) [inline astro/release-notes.md:52]:
```suggestion
- [`astro organization role list`](https://docs.astronomer.io/astro/cli/astro-organization-role-list)
- [`astro deployment user`](https://docs.astronomer.io/astro/cli/astro-deployment-user)
- [`astro deployment team`](https://docs.astronomer.io/astro/cli/astro-deployment-team)
```

File: astro/release-notes.md [hunk 7]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+See [Astro Hosted resource reference](https://docs.astronomer.io/astro/resource-reference-hosted) for more information.
+
+### Additional Deployment-level permissions customization
+
+:::publicpreview
+:::
+
+The ability to customize Deployment-level permissions with Deployment roles is now in [Public Preview](https://docs.astronomer.io/astro/feature-previews) for Enterprise tier users.
+
+In addition to this change, you can now use the new roles, **Workplace Accessor** and **Deployment Admin** to refine how you control who has access to your Deployments. See more in the [User permissions reference](https://docs.astronomer.io/astro/user-permissions#workspace-roles), [Deployment Admin](https://docs.astronomer.io/astro/user-permissions#deployment-admin), and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
```
Comment (jwitz) [inline astro/release-notes.md:45]:
```suggestion
Enterprise-tier users can additionally use the new **Workplace Accessor** and **Deployment Admin** roles to define which users have access to specific Deployments in your Workspace. See more in the [User permissions reference](https://docs.astronomer.io/astro/user-permissions#workspace-roles), [Deployment Admin](https://docs.astronomer.io/astro/user-permissions#deployment-admin), and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
```

File: astro/release-notes.md [hunk 8]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+New Asro CLI commands enable you to use the CLI to interact with and update your users' Deployment roles with the following commands:
+
+* [`astro organization role list`](https://docs.astronomer.io/astro/cli/astro-organization-role-list)
+* [`astro deployment user`](https://docs.astronomer.io/astro/cli/astro-deployment-user)
+* [`astro deployment team`](https://docs.astronomer.io/astro/cli/astro-deployment-team)
+
+
+### Export data from Reporting Dashboards with webhooks
```
Comment (jwitz) [inline astro/release-notes.md:54]:
```suggestion
### Export data from reporting dashboards with webhooks
```

File: astro/release-notes.md [hunk 9]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+:::publicpreview
+:::
+
+In addition to exporting report data with downloads or email, you can now [export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) using webhooks, so you can send your Astronomer data to your webhook endpoints such as Segment, Airtable, or Marketo.
```
Comment (jwitz) [inline astro/release-notes.md:59]:
```suggestion
In addition to exporting report data with downloads or email, you can now [export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) using webhooks. Use webhooks to send your reporting data to services such as Segment, Airtable, or Marketo.
```

File: astro/release-notes.md [hunk 10]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+Note that Webhook exports is a [Sigma beta feature](https://help.sigmacomputing.com/docs/webhook-exports) and might experience behavior changes.
+
+### Additional improvements
+
+- Added a new **Credits** tab to the [Organization Billing page](https://docs.astronomer.io/astro/manage-billing), so you can see your credit balance.
```
Comment (jwitz) [inline astro/release-notes.md:65]:
```suggestion
- You can now use the new **Credits** tab to the [Organization Billing page](https://docs.astronomer.io/astro/manage-billing) so you can see your credit balance.
```

File: astro/release-notes.md [hunk 11]
```diff
@@ -21,6 +21,51 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+- For Astro Hybrid, the maximum worker concurrency increased from `64` to `256`.
```
Comment (jwitz) [inline astro/release-notes.md:66]:
```suggestion
- On Astro Hybrid, the maximum worker concurrency on a worker queue has increased from `64` to `256`.
```

**Review by jwitz — COMMENTED** [review]
Release notes LGTM but let's hold on merging until I also review https://github.com/astronomer/docs/pull/3532/files

File: astro/manage-billing.md [hunk 12]
```diff
@@ -49,9 +53,16 @@ The **Billing** page updates every hour with your total spend in the previous co
...
-1. In the Cloud UI, click your Workspace name in the upper left corner, click **Organization Settings**, then click **Billing**. 
-2. Click **Usage**. 
+1. In the Cloud UI, click your Workspace name in the upper left corner, click **Organization Settings**, then click **Billing**.
+2. Click **Usage**.
 
 Each chart on this page shows your usage over the last 30, 60, or 90 days for each billable component type. Hover over nodes on the chart to see your total hour spend for a component on a given day. Use the dropdown menu in the upper right corner to change the timeframe for each chart.
 
 The charts are organized by component type. For a given component type, the chart shows usage for all instances of a given component type across your Organization. Therefore, it's possible to have more than 24 hours of usage for a given day. For example, if you have two **Deployment with Medium Schedulers** running for 24 hours in a day, the usage chart will show that your total usage for the day was 48 hours.
+
+## View your credits
```
Comment (jwitz) [inline astro/manage-billing.md:63]:
```suggestion
## View your credits

If you're running Astro as a trial, you can check how many complimentary credits you have left from the Astro UI.
```

File: astro/manage-billing.md [hunk 13]
```diff
@@ -49,9 +53,16 @@ The **Billing** page updates every hour with your total spend in the previous co
...
+
+1. In the Astro UI, click your Workspace name in the upper left corner, click **Organization Settings**, then click **Billing**.
+2. Click **Credits**.
+
+You can see your available credit balance and total consumption, including your credit balance when you sign up for an [Astro Trial](trial.md), to keep track of your $300 in free trial credits.
```
Comment (jwitz) [inline astro/manage-billing.md:68]:
```suggestion
```

File: astro/manage-billing.md [hunk 14]
```diff
@@ -49,9 +53,16 @@ The **Billing** page updates every hour with your total spend in the previous co
...
-2. Click **Usage**. 
+1. In the Cloud UI, click your Workspace name in the upper left corner, click **Organization Settings**, then click **Billing**.
+2. Click **Usage**.
 
 Each chart on this page shows your usage over the last 30, 60, or 90 days for each billable component type. Hover over nodes on the chart to see your total hour spend for a component on a given day. Use the dropdown menu in the upper right corner to change the timeframe for each chart.
 
 The charts are organized by component type. For a given component type, the chart shows usage for all instances of a given component type across your Organization. Therefore, it's possible to have more than 24 hours of usage for a given day. For example, if you have two **Deployment with Medium Schedulers** running for 24 hours in a day, the usage chart will show that your total usage for the day was 48 hours.
+
+## View your credits
+
```
Comment (jwitz) [inline astro/manage-billing.md:64]:
```suggestion

You can check your available credit balance and total credit consumption in the Astro UI. This includes [Astro Trial](trial.md) credits as well as any other credits you might have accrued during your time using Astro.

```

File: astro/release-notes.md [hunk 15]
```diff
@@ -21,6 +21,50 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+See [Astro Hosted resource reference](https://docs.astronomer.io/astro/resource-reference-hosted) for more information.
+
+### Additional Deployment-level permissions customization
+
+:::publicpreview
+:::
+
+The ability to customize Deployment-level permissions with Deployment roles is now in [Public Preview](https://docs.astronomer.io/astro/feature-previews) for Enterprise tier users.
+
+Enterprise-tier users can additionally use the new **Workplace Accessor** and **Deployment Admin** roles to define which users have access to specific Deployments in your Workspace. See more in the [User permissions reference](https://docs.astronomer.io/astro/user-permissions#workspace-roles), [Deployment Admin](https://docs.astronomer.io/astro/user-permissions#deployment-roles), and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
```
Comment (jwitz) [inline astro/release-notes.md:45]:
```suggestion
You can additionally use the new **Workplace Accessor** and **Deployment Admin** roles to define which users have access to specific Deployments in your Workspace. See more in [User permissions reference](https://docs.astronomer.io/astro/user-permissions#workspace-roles) and and [Create and assign custom Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles).
```

File: astro/release-notes.md [hunk 16]
```diff
@@ -21,6 +21,50 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+New Asro CLI commands enable you to use the CLI to interact with and update your users' Deployment roles with the following commands:
+
+- [`astro organization role list`](https://docs.astronomer.io/astro/cli/astro-organization-role-list)
+- [`astro deployment user`](https://docs.astronomer.io/astro/cli/astro-deployment-user)
+- [`astro deployment team`](https://docs.astronomer.io/astro/cli/astro-deployment-team)
```
Comment (jwitz) [inline astro/release-notes.md:51]:
```suggestion
```

Comment (jwitz) [reply astro/release-notes.md:51]: I feel p strongly about keeping the CLI content separate from the core feature content here, let's either backport the CLI release notes or find another way to communicate this

File: astro/release-notes.md [hunk 17]
```diff
@@ -21,6 +21,50 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Export data from reporting dashboards with webhooks
+
+:::publicpreview
+:::
+
+In addition to exporting report data with downloads or email, you can now [export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) using webhooks. Use webhooks to send your reporting data to services such as Segment, Airtable, or Marketo.
+
+Note that Webhook exports is a [Sigma beta feature](https://help.sigmacomputing.com/docs/webhook-exports) and might experience behavior changes.
```
Comment (jwitz) [inline astro/release-notes.md:60]:
```suggestion
Note that webhook exports are a [Sigma feature in beta](https://help.sigmacomputing.com/docs/webhook-exports) and might experience behavior changes.
```

**Pull request merged**